### ISSUE TITLE
Do not include ID/path in the content list view

### ISSUE THREAD

**Issue by wouterj** [symfony-cmf/sonata-phpcr-admin-integration-bundle#91]
Imo, showing the path to a content object (`/cms/content/home`) isn't usefull for admins at all. It's title (already shown), author and whether it's published ( #32 ) would be very usefull for admins.

**Comment by dbu** [comment]
agreed. users that care about the path can use the tree. and this is just a
default suggestion, people can always customize
